src/pages/VendorTasks/VendorTasks.tsx

```diff
@@ -0,0 +1,48 @@
+import { FC } from 'react'
```

Comment:
**[blocker]**

We shouldn't allow navigating to those tasks for other users, only the vendor who is the assignee of the task should be able to view the task.